roles/matrix-bridge-mautrix-whatsapp/templates/config.yaml.j2

```diff
@@ -51,79 +52,146 @@ bridge:
+    # Should another user's cryptographic identity changing send a message to Matrix?
+    identity_change_notices: false
+    # Should a "reactions not yet supported" warning be sent to the Matrix room when a user reacts to a message?
+    reaction_notices: true
+
+    portal_message_buffer: 128
+
+    # Settings for handling history sync payloads. These settings only apply right after login,
+    # because the phone only sends the history sync data once, and there's no way to re-request it
+    # (other than logging out and back in again).
+    history_sync:
+        # Should the bridge create portals for chats in the history sync payload?
+        create_portals: true
+        # Maximum age of chats in seconds to create portals for. Set to 0 to create portals for all chats in sync payload.
+        max_age: 604800
+        # Enable backfilling history sync payloads from WhatsApp using batch sending?
+        # This requires a server with MSC2716 support, which is currently an experimental feature in synapse.
+        # It can be enabled by setting experimental_features -> msc2716_enabled to true in homeserver.yaml.
+        # Note that as of Synapse 1.46, there are still some bugs with the implementation, especially if using event persistence workers.
+        backfill: false
+        # Use double puppets for backfilling?
+        # In order to use this, the double puppets must be in the appservice's user ID namespace
+        # (because the bridge can't use the double puppet access token with batch sending).
+        # This only affects double puppets on the local server, double puppets on other servers will never be used.
+        # Doesn't work out of box with this playbook
+        double_puppet_backfill: false
+        # Should the bridge request a full sync from the phone when logging in?
+        # This bumps the size of history syncs from 3 months to 1 year.
+        request_full_sync: false
+
+    user_avatar_sync: true
+    # Should Matrix users leaving groups be bridged to WhatsApp?
+    bridge_matrix_leave: true
+
+
+    # Should the bridge sync with double puppeting to receive EDUs that aren't normally sent to appservices.
     sync_with_custom_puppets: true
-    # Shared secret for https://github.com/devture/matrix-synapse-shared-secret-auth
+    # Should the bridge update the m.direct account data event when double puppeting is enabled.
+    # Note that updating the m.direct event is not atomic (except with mautrix-asmux)
+    # and is therefore prone to race conditions.
+    sync_direct_chat_list: false
+    # When double puppeting is enabled, users can use `!wa toggle` to change whether
+    # presence and read receipts are bridged. These settings set the default values.
+    # Existing users won't be affected when these are changed.
+    default_bridge_receipts: true
+    default_bridge_presence: true
+    # Servers to always allow double puppeting from
+    double_puppet_server_map: "{{ matrix_domain }}"
```

Comment:
Looking at the [example configuration](https://mau.dev/mautrix/whatsapp/-/blob/master/example-config.yaml#L136), this needs to be a map/dictionary (domain name => URL).

As an example, see the `matrix_mautrix_instagram_bridge_login_shared_secret_map` map definition. That one is a (domain => shared secret), however. I'm guessing you need `matrix_mautrix_whatsapp_homeserver_domain` and `matrix_mautrix_whatsapp_homeserver_address`:

```yaml
matrix_mautrix_whatsapp_brdige_double_puppet_server_map: "{{ {matrix_mautrix_whatsapp_homeserver_domain: matrix_mautrix_whatsapp_homeserver_address} }}"
```
or this (if it works syntax-wise):

```yaml
matrix_mautrix_whatsapp_brdige_double_puppet_server_map:
  {{ matrix_mautrix_whatsapp_homeserver_domain }}: {{ matrix_mautrix_whatsapp_homeserver_address }}
```